tests/gtests/internals/test_sdpa.cpp

```diff
@@ -737,13 +754,17 @@ class sdpa_test_t : public ::testing::TestWithParam<sdpa_dims_t> {
     }
 
 protected:
+    static dnnl::engine eng;
+    static dnnl::stream strm;
```

Comment:
Did you get any issue with OCL/SYCL tests where these might be destroyed after runtime unloading?

Reply:
Good question! Although I haven't run into these in CI nor manual testing, I've gone ahead and refactored the engine to match the test_engine we have in [gtests/main.cpp ](https://github.com/uxlfoundation/oneDNN/blob/92c54d99b8e5dbfea96111b09ecdef37ba6f155a/tests/gtests/main.cpp#L121) since I think the concern is plausible. It should now explicitly delete all resources on test fixture breakdown.